report/sections/lessons-learned-perspective/reflection.tex

```diff
@@ -0,0 +1,19 @@
+\subsection{Reflection}
+We believe we have achieved a high flow rate in our project, by fully automating the process from code to server to "customer".
+This has allowed us to work incrementally change our system, and get quick feedback from the customers and monitoring systems.
+Our automation systems also allows for easy rollback of any changes that may have caused issues.\\
+
+This way of deploying a lot instead of large deployments is very different from our usual projects. As they would usually be building the project and then releasing the entire thing at once.
+Instead we here start with a project that works and incrementally update, fix, and improve from there, while having it deployed and recieving requests.
+It's an interesting and much more reassuring way of working, as it allows us to instantly determine whether or not our changes had a positive, or negative, effect in a production enviroment.
+Which in turn provides us with a constant feedback loop that we can use to improve the system even more.
+This way of working is worth considering in later projects, though it does require some way of creating system load.\\
+
+This course emphasises risk taking and continual learning, by having us learn and use a new technology, mostly, every week.
+But also encourages trying something outside of the preset parameters.
+We've taken this to heart and tried a host of different technologies until we found the one that fit our needs. First we used Vagrant with DigitalOcean, then Terraform with Azure WebApps, and finally Kubernetes on AKS.
+In databases we've tried different technologies as well, first using an SQLite database, then moving to Azure SQL database and lastly Microsoft SQL server in Kubernetes.
+This way of working, while risky, has allowed us to learn new technologies and improve the deployment and hosting process, which provides value both to us as developers, but also to our customers who get faster updates and a faster service.
+It's a way of working we have enjoyed and will try to use in later projects. It does, though, seem like a work method that mostly makes sense in projects where the system is already in a production ready state and running. Then you can start experimenting with technologies and new methods.
```

Comment:
Is it possible to rephrase or split this sentence? The "way of working" thread is maybe a little long? I at least found it a little hard near the end specifically at 

"It does, though, seem like a work... "